modules/auxiliary/gather/browser_sop_bypass.rb

```diff
@@ -0,0 +1,53 @@
+##
```

Comment:
Why is this a gather aux module if this is supposed to be an SOP bypass? What does it gather?

Reply:
Update, this seems to want to gather usernames and passwords for Google, but doesn't store them anywhere.